[Tidy] Improve appearance of README on PyPI

manuelkonrad opened the pull request:
## Description

* In README.md, changed default logo to the light mode logo which hopefully fixes the appearance on PyPI.
* Added logo to Vizro-AI README.
* Since GIFs are also broken on PyPI (although I saw working examples when markdown syntax is used), I placed them in a `picture` element. Not sure if it helps. ~~If not, you can provide an alternative PNG in the `img` element (will serve as fallback).~~ Added fallback PNGs.
* Resolves #380   

## Screenshot

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

    - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
    - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
    - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
    - I have not referenced individuals, products or companies in any commits, directly or indirectly.
    - I have not added data or restricted code in any commits, directly or indirectly.

huong-li-nguyen commented on vizro-ai/README.md:
```diff
@@ -1,10 +1,27 @@
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/Vizro_Github_Banner_Dark_Mode.png">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/Vizro_Github_Banner_Light_Mode.png">
+  <img alt="Vizro logo" src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/Vizro_Github_Banner_Light_Mode.png" width="250">
+</picture>
+
+</div>
+
+<br>
+
 <h1 align="center">Vizro-AI</h1>
 
 <div align="center" markdown="1">
 
 [![Python version](https://img.shields.io/badge/python-3.9%20%7C%203.10%20%7C%203.11%20%7C%203.12-blue.svg)](https://pypi.org/project/vizro/) [![PyPI version](https://badge.fury.io/py/vizro_ai.svg)](https://badge.fury.io/py/vizro_ai) [![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/mckinsey/vizro/blob/main/LICENSE.md) [![Documentation](https://readthedoc ...
 
-<img src="./docs/assets/readme/readme_animation.gif" alt="Gif to demonstrate vizro-ai">
+<picture>
+  <source srcset="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/toolkit_vizro_ai.gif">
+  <img alt="Gif to demonstrate Vizro-AI" src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/toolkit_vizro_ai.gif" width="600">
+</picture>
 
```
Good catch! 👍 Given that these images are duplicated, I think we can delete the entire folder: https://github.com/mckinsey/vizro/tree/main/vizro-ai/docs/assets/readme

And then you would have to replace the link to the image here as well: https://github.com/mckinsey/vizro/blob/main/vizro-ai/docs/index.md

manuelkonrad replied:
Would it be fine if the image is not hosted by RTD anymore? Otherwise we could do it the other way around and use the GIF from the assets folder in the README.

huong-li-nguyen replied:
I think the changes you've made are correct 👍. We don't need to duplicate these images in different folders. The docs look fine with the changes, so all good:  https://vizro--969.org.readthedocs.build/projects/vizro-ai/en/969/

huong-li-nguyen reviewed (approved):
Hey @manuelkonrad - thanks for taking on another issue! 🚀 
This looks good to me. I have a few minor comments, but it's ready to merge from my side 👍 I'll leave the final review to @stichbury or @Joseph-Perkins .

stichbury commented:
> Is this what you see when you click on [this link](https://github.com/manuelkonrad/vizro/blob/tidy/improve-pypi-logos/README.md)? For me, it looks fine 👍

That does look fine. I'm looking at the changed file in the PR though, and that doesn't: https://github.com/mckinsey/vizro/blob/ac400c876856f04bef519d29faf8b1eef857bb6c/README.md

Could be just because it's on a fork I guess.

manuelkonrad commented:
As suggested by @huong-li-nguyen, I also changed the image reference in `vizro-ai/docs/index.md` and removed the redundant `gif`. Therefore, the docs will have a direct link to the `gif` hosted on github.

stichbury reviewed (approved):
Thank you for the contribution @manuelkonrad!

antonymilne commented:
Ahhh this issue has bugged me for ages! Thank you very much for your contribution @manuelkonrad.

manuelkonrad commented:
> Ahhh this issue has bugged me for ages! I'm so pleased to see it resolved. Thank you very much for your contribution @manuelkonrad.

@antonymilne Let's see after the next release whether the PyPI page actually looks better (or you upload to Test PyPI in the meantime).

BTW, really amazing how fast you iterate on PRs!

huong-li-nguyen commented:
> @antonymilne Let's see after the next release whether the PyPI page actually looks better (or you upload to Test PyPI in the meantime).

We just released - it looks great! 🚀  https://pypi.org/project/vizro/
